src/pages/components/text-input/usage.mdx

```diff
@@ -132,99 +152,365 @@ reflect the length of the content you expect the user to enter.
+toggle on the character visibility by clicking the view icon on the far right of
+the input field. When using a password input be sure to provide detailed helper
+text listing any requirements related to the data format, such as types of
+character allowed or date structure.
 
 <Row>
 <Column colLg={8}>
 
-![An example of a text input.](images/text-input-header-usage.png)
+![Example of a password input](images/text-input-usage-password.png)
 
 </Column>
 </Row>
 
-#### Accessibility best practices for labels
+## Text area
 
-- Labels must be visible when an input gets focus.
-- Labels must be announced to the screen reader on focus.
-- Ensure the helper text that appears under an input is read when an assistive
-  technology user stops at an input using ARIA.
+Use a text area when the expected user input is more than a few words and could
+span multiple lines. It is commonly used for features like user commentary or
+descriptions. It supports all the same states and functionality as text input
+except for the password functionality. Text area has several unique
+functionalities not included in the default text input, like the resize handle
+and character counter.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a text area](images/text-input-usage-2.png)
 
-### Placeholder text
+</Column>
+</Row>
 
-Placeholder text provides hints or examples of what to enter. Placeholder text
-disappears after the user begins entering data into the input and should not
-contain crucial information. Use sentence-style capitalization, and in most
-cases, write the text as a direct statement without punctuation.
+### Resize handle
+
+Included by default in text area is the `resize` handle. It allows a user the
+manipulate the field height by making it longer or shorter. The resize handle
+has no effect on the width of the text area container, it only effects the
+height. If the user makes the field size shorter than the content inside the
+field then a vertical scroll will become available.
 
 <Row>
 <Column colLg={8}>
 
-![Placeholder text example](images/text-input-usage-1.png)
+![Example showing the resize handle in text area](images/text-area-usage-resize.png)
 
 </Column>
 </Row>
 
-### Helper text
+### Character counter
 
-Helper text is pertinent information that assists the user in completing a
-field. Helper text is always available when the input is focused and appears
-underneath the label. Use sentence-style capitalization, and in most cases,
-write the text as full sentences with punctuation.
+A character counter can be added to text area to indicate both the number of
+characters being entered and the total number of characters allowed. Once the
+max number of characters is reached the text area should should prevent the user
+from entering any additional character and provide messaging to the user that a
+limit has been met.
 
 <Row>
 <Column colLg={8}>
 
-![Helper text example](images/text-input-usage-3.png)
```

Comment:
- This image has a typo. It should be `charged` not charge.

<img width="543" alt="Screen Shot 2022-08-26 at 3 23 30 PM" src="https://user-images.githubusercontent.com/43969356/186977053-312dbc06-6fe7-4a57-8249-eb2b43eb37b3.png">